CRM/Utils/HttpClient.php

```diff
@@ -98,7 +98,13 @@ public function fetch($remoteFile, $localFile) {
     }
     curl_setopt($ch, CURLOPT_FILE, $fp);
 
-    curl_exec($ch);
+    if ($this->isRedirectSupported()){
+      curl_exec($ch);
+    }
+    else{
+      $this->curl_exec_follow($ch);
```

Comment:
Yes you are right .. I replaced it and retested and it's working fine in both cases